article/_posts/2019-11-27-SHA_RNN.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: review
+title: "Single Headed Attention RNN: Stop Thinking With Your Head"
+tags: deep-learning NLP RNN
+author: "Antoine Théberge"
+cite:
+    authors: "Stephen Merity"
+    title:   "Single Headed Attention RNN: Stop Thinking With Your Head"
+    venue:   "Arxiv"
+pdf: "https://arxiv.org/pdf/1911.11423.pdf"
+---
+
+
+# Highlights
+
+- Simple RNN architecture provides (almost) state of the art results
+- "Tokenization attacks" make comparing results difficult
+
+# Preface
+
+The currently reviewed article requires a very good understanding of NLP, attention and the transformer module. These two articles ([1.](https://jalammar.github.io/visualizing-neural-machine-translation-mechanics-of-seq2seq-models-with-attention/) [2.](http://jalammar.github.io/illustrated-transformer/)) provide a good summary of these concepts.
+
+# Introduction
+
+State of the art (SotA) NLP architectures are massive and require enormous resources to train, which makes meaningful research hard to do for smaller research teams. The author painfully introduces its own NLP architecture, based on a previous paper of his <sup>[1]</sup>, called SHA-RNN.
```

Comment:
his own

Reply:
did you know that our markdown flavor supports references? I just added an example in the template on master

Reply:
Fixed it, thanks !